### PR TITLE
[DOCFIX] Add glue udb into docs

### PR DESCRIPTION
**Pull request by HelloHorizon**
Add the glue udb to docs.

### CODE REVIEW

File: docs/en/operation/User-CLI.md [hunk 1]
```diff
@@ -1152,6 +1152,12 @@ Here are the additional properties possible for the `-o` options:
   * `catalog.db.sync.threads`: number of parallel threads to use to sync with the UDB. If too large,
   the sync may overload the UDB, and if set too low, syncing a database with many tables make take
   a long time. The default is `4`.
+  * `udb-glue.aws.<MOUNT_PROPERTY>`: specify the mount option for the Glue udb, the mount options
```
Comment (gpang) [inline docs/en/operation/User-CLI.md:1155]: maybe we should have 2 sub-sections, one for the hive udb type, and one for the glue udb type. Then, in each section, we can talk about the specific options. I don't think hive has any hive specific options, so the glue sub-section would discuss the glue-specific ones.

Comment (HelloHorizon) [reply docs/en/operation/User-CLI.md]: I have divided the doc into two part with Hive and Glue, please check it.

File: docs/en/operation/User-CLI.md [hunk 2]
```diff
@@ -1169,6 +1171,31 @@ This command will attach the database `hive_db_name` (of type `hive`) from the U
...
+  are as follows:
+    * `region`: the glue aws region
+    * `catalog.id`: the aws catalog id
+    * `accesskey`: the aws access key id
+    * `secretkey`: the aws secret key
+
+You can supply the mount options for the `glue` as follows:
+
+```console
+$ ./bin/alluxio table attachdb --db alluxio_db_name Glue null glue_db_name \
```
Comment (gpang) [inline docs/en/operation/User-CLI.md:1186]: should `Glue` be `glue`?

File: docs/en/operation/User-CLI.md [hunk 3]
```diff
@@ -1169,6 +1171,31 @@ This command will attach the database `hive_db_name` (of type `hive`) from the U
...
 and when paths are mounted for `s3a://bucket2`, the mount option `aws.accessKeyId=123` will be used.
 
+### Glue UDB
+For `glue` udb type, there are some additional properties with the `-o` options:
+  * `udb-glue.aws.<MOUNT_PROPERTY>`: specify the mount option for the Glue udb, the mount options
+  are as follows:
+    * `region`: the glue aws region
+    * `catalog.id`: the aws catalog id
+    * `accesskey`: the aws access key id
+    * `secretkey`: the aws secret key
```
Comment (gpang) [inline docs/en/operation/User-CLI.md:1181]: I think this is how the properties should be called

```suggestion
  * `udb-glue.<UDB_PROPERTY>`: specify the UDB options for the Glue UDB. The options
  are as follows:
    * `aws.region`: the glue aws region
    * `aws.catalog.id`: the aws catalog id
    * `aws.accesskey`: the aws access key id
    * `aws.secretkey`: the aws secret key
```

File: docs/en/operation/User-CLI.md (hunk 2 above)
Comment (HelloHorizon) [reply docs/en/operation/User-CLI.md]: Fixed.

File: docs/en/operation/User-CLI.md (hunk 3 above)
Comment (HelloHorizon) [reply docs/en/operation/User-CLI.md]: It makes sense, Fixed.

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
`alluxio.client.fs.FileOutStreamAsyncWriteIntegrationTest.asyncWriteNoEvictBeforeBlockCommit` is known to be flaky, and this is a docfix.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**